runtime/nls/j9vm/j9vm.nls

```diff
@@ -2316,6 +2316,7 @@ J9NLS_VM_CRIU_CRAC_INCOMPATIBLE_SETTING.user_response=Remove one of the options
 
 J9NLS_VM_CRIU_J9_GET_PROCESS_START_TIME_FAILURE=j9sysinfo_get_process_start_time failed with errno=%zi
 # START NON-TRANSLATABLE
+J9NLS_VM_CRIU_J9_GET_PROCESS_START_TIME_FAILURE.sample_input_1=-34
```

Comment:
I wondered about that when reviewing https://github.com/eclipse-openj9/openj9/pull/18680, but I saw other (integer) template parameters that didn't have samples, so I assumed it was unnecessary for integer formats. See entries at lines 2200, 2207, 2214 to list just the first few. If a sample is needed, it should be added for all.

Reply:
Pushed another commit to resolve the other missing samples.